packages/common/src/polyfills/sdk-shim/withModalProvider.tsx

```diff
@@ -11,14 +13,25 @@ import { ModalProvider } from '../console-dynamic-plugin-sdk/app/modal-support/M
  * the modal provider context.
  */
 export const withModalProvider = <T extends object>(Component: React.ComponentType<T>): React.FC<T> => {
-  function ModalProviderHoc(props) {
+  const ModalProviderHocPolyfill = (props) => {
     return (
       <ModalProvider>
         <Component {...props} />
       </ModalProvider>
     );
   }
 
+  const ModalProviderHocSDK = (props) => {
```

Comment:
can we do an early return with the component?  i.e.
```
if(!IS_CONSOLE_VERSION_4_11) {
   return Component;
}

```

Reply:
nice, trying now